packages/recommendations-js/src/frequentlyBoughtTogether.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx h */
+import {
+  getFrequentlyBoughtTogether,
+  GetFrequentlyBoughtTogetherProps,
+  GetRecommendationsReturn,
+} from '@algolia/recommendations-core';
+import {
+  createFrequentlyBoughtTogetherComponent,
+  FrequentlyBoughtTogetherProps,
+} from '@algolia/recommendations-vdom';
+import { createElement, Fragment, h, render } from 'preact';
+import { useEffect, useState } from 'preact/hooks';
+
+import { getHTMLElement } from './getHTMLElement';
+import { EnvironmentProps } from './types';
+import { useAlgoliaAgent } from './useAlgoliaAgent';
+
+const UncontrolledFrequentlyBoughtTogether = createFrequentlyBoughtTogetherComponent(
+  {
+    createElement,
+    Fragment,
+  }
+);
+
+function useFrequentlyBoughtTogether<TObject>(
+  props: GetFrequentlyBoughtTogetherProps<TObject>
+) {
+  const [result, setResult] = useState<GetRecommendationsReturn<TObject>>({
```

Comment:
```suggestion
  const [result, setResult] = useState<GetRecommendationsResult<TObject>>({
```

I feel like this is named more naturally, since it's a network response/result, not really a return

Reply:
Addressed in 38f20bc3b64e1dd097b7148a284ba3a6b201c293.